evm/client/rpc_client.go

```diff
@@ -1227,7 +1252,17 @@ func (r *RPCClient) IsSyncing(ctx context.Context) (bool, error) {
 	var syncProgress *ethereum.SyncProgress
 	start := time.Now()
 	var err error
-	if http != nil {
+	if r.isChainType(chaintype.ChainTron) {
+		var isSyncing bool
+		err = r.wrapHTTP(http.rpc.CallContext(ctx, &isSyncing, "eth_syncing"))
```

Comment:
I'm assuming the Tron node doesn't return the `syncProgress` data and `eth_syncing` always returns a bool? Why doesn't the underlying geth method work?
https://github.com/ethereum/go-ethereum/blob/master/ethclient/ethclient.go#L349